extensions/wikia/WikiaHomePage/WikiaHomePage.setup.php

```diff
@@ -15,11 +15,11 @@
 $wgExtensionCredits['other'][] = array(
 	'name'				=> 'WikiaHomePage',

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "WikiaHomePage" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)